### ISSUE TITLE
Vertical screen button images are upside-down and small

### ISSUE THREAD

**Issue by AndyHazz** [arcadeforge/regamebox#65]
Using vertical rotation, button images are rotated upside down before the game loads. I've tried with ORIENTATION_V="right" and "left" in regamebox.conf but that makes no difference.

...  I found changing 90 to 270 on line 357 of autostart.sh seems to resolve the upside-down issue (a restart was needed after making the change to have any effect).

... then changing `"$h" "$w" 2000 270`  to `"$h" "$h" 2000 270` shows the images at a better size on screen - though still not perfect - I tried a few different things and couldn't get it spot on (seems shifted to the right and not using as much space as it could).

Ideally there would be a way to allow  true 240x320 pngs for full screen vertical control images.

**Comment by arcadeforge** [comment]
Hmm. I have tested in 3.2.2 and it looks OK.

**Comment by AndyHazz** [comment]
Odd - still upside down for me after 3.2.2 online update - I'll try a fresh image and patch before plugging in an empty USB and see where that gets me, maybe I messed up something.

**Comment by AndyHazz** [comment]
ok, fresh setup and I noticed the default vertical orientation is upside down for me. I used flip screen in the options menu and it flips the menu/game screens but not the button images. So, it looks like there's something to fix there, if you can get the button images to respect flip screen setting?

In either orientation the button image seems smaller than the available screen width - as we're squashing a horizontal image into a vertical screen every pixel counts! - so if you do get chance to look into the button flip it'd be great if you could also review options to get a larger button image to display on the vertical layout?

**Comment by arcadeforge** [comment]
ahh, ok, so you have a flipped screen setup and this is not taken into account by the show_image / show_button script.

**Comment by AndyHazz** [comment]
Correct, that's the main issue here - but also (maybe this should be a separate issue on github) that even when the right way up, the button image doesn't fill the screen width in a vertical setup.

I think a 320x240px button image should be scaled down to 240x180px in a vertical rotation? However, it looked like the button image is more like 180x135px in vertical mode (roughly, I'm guessing) ... noticeably smaller than it should be.

**Comment by arcadeforge** [comment]
yeah, possible. Perhaps I wanted to avoid stretching the image back in the days :)

**Comment by AndyHazz** [comment]
I don't mean you should stretch anything - attaching a photo so you can see what I mean (my monitor isn't rotated right now but I have put regamebox in vertical mode and turned my brightness way up so you can see where the button image is sitting) - it could be much bigger without stretching anything, it's not filling the full width for some reason:

![PXL_20210527_204420351 MP](https://user-images.githubusercontent.com/517731/119895006-34816800-bf35-11eb-8f6c-3896f78c8bab.jpg)
